source/common/http/conn_manager_impl.cc

```diff
@@ -1171,6 +1178,7 @@ void ConnectionManagerImpl::ActiveStream::decodeMetadata(MetadataMapPtr&& metada
 }
 
 void ConnectionManagerImpl::ActiveStream::disarmRequestTimeout() {
+  state_.downstream_request_read_complete_ = true;
```

Comment:
filter_manager_.remoteDecodeComplete() ?

Reply:
Awesome! If I sync with the main branch I should have seen it...

Let me update. Thanks！